docs/source/api/deliveryservices_hostname_name_sslkeys.rst

```diff
@@ -70,9 +70,12 @@ Response Structure
 :hostname:        The hostname generated by Traffic Ops that is used as the common name when generating the certificate - this will be an :abbr:`FQDN (Fully Qualified Domain Name)` for DNS :term:`Delivery Services` and a wildcard URL for HTTP :term:`Delivery Services`
 :organization:    An optional field which, if present, contains the organization entered by the user when generating certificate\ [#optional]_
 :state:           An optional field which, if present, contains the state entered by the user when generating certificate\ [#optional]_
-:version:         The version of the certificate record in Traffic Vault
+:version:         An integer that defines the "version" of the key - which may be thought of as the sequential generation; that is, the higher the number the more recent the key
 
-.. code- block:: http
+:expiration:      The expiration date of the certificate for the :term:`Delivery Service` following RFC3339 internet date/time format YYYY-MM-DDTHH:mm:ssZ
```

Comment:
I'd feel bad asking you to change this yet again, and this gets the point across, so feel free to ignore this, but you didn't need to specify the form yourself, you can just link to the RFC. Which is easy to do with the built-in `rfc` role:
```rst
:expiration: The expiration date of the certificate for the :term:`Delivery Service` in :rfc:`3339` format
```

Reply:
oh thats slick! i like it!
done